weblogic-azure-vm/arm-oraclelinux-wls-admin/pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.oracle.weblogic.azure</groupId>
     <artifactId>arm-oraclelinux-wls-admin</artifactId>
-    <version>1.0.33</version>
+    <version>1.0.34</version>
```

Comment:
https://github.com/oracle/weblogic-azure/pull/188 POM version upgrades Merge this first

Then Ed or Zheng will rebase this PR on top of 188 and increment the POM versions yet again.